scripts/run_latest_build.sh

```diff
@@ -74,23 +74,32 @@ oc new-project ansible-service-broker
 #
 TEMPLATE_URL="https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml"
```

Comment:
You don't have to make this change..but I would appreciate it, make this value override-able `TEMPLATE_URL=${TEMPLATE_URL:-"..."}`.

Reply:
@djzager You will need to use the broker image from the PR when running run_latest

Reply:
Yeah. That is what I was holding wrong 😎. Still playing with it.